openid4vc-rest-api.json

```diff
@@ -0,0 +1,77 @@
+{
+    "clientId": "openid4vc-rest-api",
+    "name": "",
+    "description": "",
+    "rootUrl": "",
+    "adminUrl": "",
+    "baseUrl": "",
+    "surrogateAuthRequired": false,
+    "enabled": true,
+    "alwaysDisplayInConsole": false,
+    "clientAuthenticatorType": "client-secret",
+    "secret": "uArydomqOymeF0tBrtipkPYujNNUuDlt",
+    "redirectUris": [
```

Comment:
It is not a good idea to have this secret here. Can you find a way to set it through env vars and configure on the repository?